.github/workflows/main.yml

```diff
@@ -1,61 +1,54 @@
-﻿# Uses https://game.ci/docs/github/getting-started
-name: Actions
-
+﻿name: Actions
 on: [push]
 
+env:
+  BUILD_OUTPUT_FOLDER: 'build/WebGL/WebGL'
+  PRODUCTION_BUILD_ARTEFACT_NAME: 'viewer'
+  DEVELOPMENT_BUILD_ARTEFACT_NAME: 'viewer-development'
+
 jobs:
   build:
-    name: Build my project ✨
-    runs-on: ubuntu-latest
-    
-    permissions:
-      contents: read
-      packages: write
+    name: '🗺️ Build Netherlands3D Viewer'
+
+    # only run this when a commit was pushed to the main branch
+    if: github.ref_name == 'main'
 
+    permissions: { contents: read }
+    outputs:
+      buildVersion: steps.buildUnity.outputs.buildVersion 
+
+    runs-on: ubuntu-latest
     steps:
-      # Clean space
-      - name: Free Disk Space (Ubuntu)
+      - name: 'Free Disk Space'
+        # see: https://github.com/jlumbroso/free-disk-space  
         uses: jlumbroso/free-disk-space@main
         with:
-          # this might remove tools that are actually needed,
-          # if set to "true" but frees about 6 GB
           tool-cache: false
-          
-          # all of these default to true, but feel free to set to
-          # "false" if necessary for your workflow
           android: true
           dotnet: true
           haskell: true
-          large-packages: true
+          large-packages: false
```

Comment:
This is disabled because it speeds up this build step significantly, and the cleared disk space is not perse needed